HighwayPatrol.cpp

```diff
@@ -28,7 +30,7 @@ void HighwayPatrol::pullOver( Vehicle* v, bool willArrest, Highway* h )
     if( willArrest )
     {
         //print the vehicle type in this std::cout between "THE [" and "] PULL". 
-        std::cout << name << ": YOU IN THE [ " << " ] PULL OVER AND SHOW YOUR HANDS" << std::endl;
+        std::cout << name << ": YOU IN THE [ " << typeid(*v).name()/*v->name*/ << " ] PULL OVER AND SHOW YOUR HANDS" << std::endl;
```

Comment:
clever, but no. 
use `if/else if` and casting to figure out the vehicle type.
then populate a string with that type, and put the string between the `[ ]`
- [ ] revise